packages/generator-one-app-module/generators/app/templates/base-module/src/components/ModuleContainer.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ModuleRoute from 'holocron-module-route';
+import csp from './csp';
+
+const <%=moduleNamePascal%> = () => (
+  <div>
+    <h1>Welcome to One App!</h1>
+  </div>
+);
+
+<%=moduleNamePascal%>.childRoutes = () => ([
```

Comment:
Do we have a link about `childRoutes` that we could add here?

Reply:
updated in updated in 1e2900f